confmap/provider/s3mapprovider/README.md

```diff
@@ -0,0 +1,14 @@
+What is this new component s3mapprovider?
+- An implementation of ConfigMapProvider for Amazon S3 (s3mapprovider) allows OTEL Collector the ability to load configuration for itself by fetching and reading config files stored in Amazon S3.
+
+How this new component s3mapprovider works?
+- It will be called by ConfigMapResolver to load configurations for OTEL Collector.
+- By giving a config URI starting with prefix 's3://', this s3mapprovider will be used to download config files from given S3 URIs, and then used the downloaded config files to deploy the OTEL Collector.
+- In our code, we check the validity scheme and string pattern of S3 URIs. And also check if there are any problems on config downloading and config deserialization.
```

Comment:
```suggestion
## Summary

This package provides a `ConfigMapProvider` implementation for Amazon S3 (`s3mapprovider`) that allows the  Collector the ability to load configuration by fetching and reading config objects stored in Amazon S3.

## How it works
- It will be called by `ConfigMapResolver` to load configuration for the Collector.
- By giving a config URI starting with prefix `s3://`, this `s3mapprovider` will be used to download config objects from the given S3 URIs, and then use the downloaded configuration during Collector initialization.
```

This component won't always be new, so we should avoid referring to it as such in this document.  Made some other tweaks to clarify the content.

Reply:
Fixed! Thanks!